licenses/models.py

```diff
@@ -1,2 +1,126 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.utils.translation import gettext_lazy as _
 
-# Create your models here.
+
+User = get_user_model()
+
+MAX_TITLE_LENGTH = 255
+
+
+class Category(models.Model):
+    """
+    Model representing a category.
+
+    Each License has a category.
+    """
+
+    name = models.CharField(
+        _('Category'),
+        blank=False,
+        null=True,
+        max_length=255
+    )
+
+    def __str__(self) -> str:
+        """Represent category by its String."""
+        return self.name
+
+    class Meta:
+        """Defines the plural name."""
+
+        verbose_name_plural = _('Categories')
```

Comment:
Please also add the `verbose_name` to make it all defined in one place.
```suggestion
        verbose_name = _('Category')
        verbose_name_plural = _('Categories')
```